modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java

```diff
@@ -334,7 +342,8 @@ protected void beforeTestsStarted() throws Exception {
      * @throws Exception If failed.
      */
     protected void afterTestsStopped() throws Exception {
-        // No-op.
+        //restoring page handler wrapper
+        BPlusTree.pageHndWrapper = regularPageHndWrapper == null ? ((tree, hnd) -> hnd) : regularPageHndWrapper;
```

Comment:
Seems, regularPageHndWrapper is never null here.
If BPlusTree.pageHndWrapper was null in beforeTestsStarted, then null value will not be restored.

Reply:
Do we really need change pageHndWrapper for all tests (in GridAbstractTest)?